### PR TITLE
setup protractor configuration

### PR DESCRIPTION
**Pull request by jalvar53**
Setting up protractor config, gitignore, licence, dependencies files and first dummy test

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    "practice",
+    "protractor"
+  ],
+  "author": "Jose Luis Alvarez Herrera - joseluh_1@hotmail.com (jalvar53)",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/jalvar53/protractor-workshop-2017/issues"
+  },
+  "homepage": "https://github.com/jalvar53/protractor-workshop-2017#readme",
+  "dependencies": {
```
Comment (luisbolanoss) [inline package.json]: protractor should be a devdependency

**Review by luisbolanoss — CHANGES_REQUESTED** [review]
Only one comment

File: README.md [hunk 2]
```diff
@@ -1 +0,0 @@
-# protractor-workshop-2017
```
Comment (germandavid85) [inline README.md]: don't remove the README file, it is a good practice to have it, later on you can add more information to it with a description of the project, steps to executed, tools, etc

**Review by germandavid85 — CHANGES_REQUESTED** [review]

**Review by luisbolanoss — APPROVED** [review]
LGTM, review the German`s comment

**Review by germandavid85 — APPROVED** [review]

**Pull request merged**